creator-node/package.json

```diff
@@ -20,6 +20,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@audius/libs": "1.2.93",
+    "@solana/web3.js": "^1.37.1",
```

Comment:
let's make this consistent and pin like we do in libs https://github.com/AudiusProject/audius-protocol/blob/master/libs/package.json#L40